docs/content/tutorials/tutorial-a-first-look-at-druid.md

```diff
@@ -74,6 +74,8 @@ You should see a bunch of files:
 * run_example_server.sh
 * run_example_client.sh
 * LICENSE, config, examples, lib directories
+* druid_extensions (This folder contains all the extensions that could be loaded by Druid. Note that extension `mysql-metadata-storage` is packaged in a separate tarball that can be downloaded from [here](http://druid.io/downloads.html). See [Including Extensions](../operations/including-extensions.html) for more information about loading extensions.
```

Comment:
Did we adjust the examples to make sure we didn't load all the extensions by default? It might cause some weirdness if multiple metadata stores or other incompatible extensions get loaded at the same time.